dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -234,3 +234,7 @@ scheme.incorrect.error.useraction=The scheme type must be http, https, ws, or ws
 upgradeHttpToWebSocket.failed=CWWKH0055E: The jakarta.websocket.server.ServerContainer#upgradeHttpToWebSocket call failed due to {0}. 
 upgradeHttpToWebSocket.failed.explanation=The server could not upgrade the HTTP or HTTPS request to the ws/wss protocol.
 upgradeHttpToWebSocket.failed.useraction=Review the error to identify why the request failed to upgrade.
+
+maxmessagesize.exceeded=CWWKH0056E: The maxMessageSize for method {0} exceeded Integer.MAX_VALUE for class {1}.
```

Comment:
Suggested change to use the "the _variable_ noun" format:

The maxMessageSize for the {0} method exceeds Integer.MAX_VALUE for the {1} class.

This rewrite assumes that the message is returned if the maxMessageSize setting exceeds Integer.MAX_VALUE. If the message is returned when the method size exceeds the maximum, then the message would be clearer as:

The size of the {0} method exceeds Integer.MAX_VALUE for the {1} class.